### ISSUE TITLE
Crash with Information Screen & Hwyla/Waila

### ISSUE THREAD

**Issue by ZXQWAS** [McJtyMods/RFToolsPower#16]
I got this crash repeating (saved by vanillafix) many times while looking at a screen, or more likely, power cell nearby that it was connected to. After a server restart I didn't have it anymore. I suspect this was caused by having 2 screens on the same multiblock powercell. Or rather 1 of them on a fluxduct that comes out of that powercell (screen also directly above the power cell).
[crash-2018-09-30_03.41.32-server.txt](https://github.com/McJtyMods/RFToolsPower/files/2431599/crash-2018-09-30_03.41.32-server.txt)

**Comment by Pultex** [comment]
Yes this seem to happen on the Skyfactory 4 mod pack when only one screen is used in any use.
It is random and seems to shutdown the server.
Not sure why as there is no indication of something wrong, going to a new dimension seems to make it happen more often.

**Comment by McJty** [comment]
I will fix this. Completely forgot about this
